framework/validators/IpValidator.php

```diff
@@ -354,14 +355,14 @@ private function validateSubnet($ip)
             return [$this->message, []];
         }
 
-        if ($this->getIpVersion($ip) == 6) {
+        if ($this->getIpVersion($ip) === IpHelper::IPV6) {
             if ($cidr !== null) {
-                if ($cidr > static::IPV6_ADDRESS_LENGTH || $cidr < 0) {
```

Comment:
Should constants be deprecated?

Reply:
done

Reply:
I think a deprecation is more than enough. BC break in this case is overkill.

Reply:
It's a small break and I don't think a lot of people relied on internal constants of IpValidator

Reply:
I do agree about internals. But the same applies to many framework areas (for example, `yii\db`) and it's a dangerous road I would not step onto.
IMHO, if we make an important feature everyone asks for years or fixing a long story bug, it's okay. But when we add a new helper, there's no need to rush. :)